stable/pachyderm/README.md

```diff
@@ -0,0 +1,142 @@
+
+| Parameter                | Description           | Default           |
+|--------------------------|-----------------------|-------------------|
+| `credentials`            | Backend credentials   | ""                |
+
+
+Based on the storage credentials used, fill in the corresponding parameters for your object store.
+
+-	The `local` installation will deploy Pachyderm on your local Kubernetes cluster (i.e: minikube) backed by your local storage unit. 
+
+-	With `S3 endpoint` credentials (such as Minio credentials), these are the configurable parameters:
+
+| Parameter                | Description           | Default           |
+|--------------------------|-----------------------|-------------------|
+| `s3.accessKey`           | S3 access key         | `""`              |
+| `s3.secretKey`           | S3 secret key         | `""`              |
+| `s3.bucketName`          | S3 bucket name        | `""`              |
+| `s3.endpoint`            | S3 endpoint           | `""`              |
+| `s3.secure`              | S3 secure             | `"0"`             |
+| `s3.signature`           | S3 signature          | `"0"`             |
+
+
+-	With `Google Cloud` credentials, you must define your `GCS bucket name`:
+
+| Parameter                | Description           | Default           |
+|--------------------------|-----------------------|-------------------|
+| `google.bucketName`      | GCS bucket name       | `""`              |
+
+
+-	On `Amazon Web Services`, please set the next values:
+
+| Parameter                | Description           | Default           |
+|--------------------------|-----------------------|-------------------|
+| `amazon.bucketName`      | Amazon bucket name    | `""`              |
+| `amazon.distribution`    | Amazon distribution   | `""`              |
+| `amazon.id`              | Amazon id             | `""`              |
+| `amazon.region`          | Amazon region         | `""`              |
+| `amazon.secret`          | Amazon secret         | `""`              |
+| `amazon.token`           | Amazon token          | `""`              |
+
+
+-	As for `Microsoft Azure`, you must specify the following parameters:
+
+| Parameter                | Description           | Default           |
+|--------------------------|-----------------------|-------------------|
+| `microsoft.container`    | Container             | `""`              |
+| `microsoft.id`           | Account name          | `""`              |
+| `microsoft.secret`       | Account key           | `""`              |
+
+
+How to install the chart
+------------------------
+
+We strongly suggest that the installation of Pachyderm should be performed in its own namespace. The default installation will deploy Pachyderm on your local Kubernetes cluster:
+
+```console
+$ helm install --name my-release stable/pachyderm
```

Comment:
Adding `--namespace pachyderm` would be good to illustrate the above recommendation.